pom.xml

```diff
@@ -1,13 +1,28 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
   <parent>
     <artifactId>fcrepo</artifactId>
     <groupId>org.fcrepo</groupId>
     <version>4.3.1-SNAPSHOT</version>
   </parent>
-  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.fcrepo.transform</groupId>
+  <artifactId>fcrepo-transform</artifactId>
+  <packaging>bundle</packaging>
+
+  <version>4.3.1-SNAPSHOT</version>
+
+  <name>Fedora Repository RDF Transformations Module</name>
+  <description>The Fedora Commons repository transformations module: Provides facilities to use SPARQL or LDPath to tranform repository RDF into desired forms.</description>
+
   <properties>
     <ldpath.version>3.2.1</ldpath.version>
+    <fcrepo.version>${project.version}</fcrepo.version>
+    <jersey.version>2.13</jersey.version>
```

Comment:
Since you are parenting off of org.fcrepo:fcrepo, you should not be redefining:
`jersey.version`, `spring.version`, and `logback.version`.
Please remove those three line items.